spark_log_parser/__init__.py

```diff
@@ -1,2 +1,8 @@
 """Tools for providing Spark event log"""
-__version__ = "0.3.0"
+import logging
+
+logging.basicConfig()
+logging.root.setLevel(logging.INFO)
```

Comment:
I'm not 100% what the "idiomatic" place is to set up logging in a Python app/library, but doing this in the root module __init__ seems pretty bullet-proof to me...